File: Ryhmätyö_kokeiluja_JAVALLA/Darts_peli/src/dartsPeli/Pelaaja.java
package dartsPeli;



public class Pelaaja {

    private String pelaajaNimi;
    private int pelaajanPisteet;

    public Pelaaja(String nimi, int pisteet){
        this.pelaajaNimi = nimi;
        this.pelaajanPisteet = pisteet;
    }

    public String getPelaajaNimi() {
        return pelaajaNimi;
    }


    @Override
    public String toString() {
        return pelaajaNimi + " (" + pelaajanPisteet + " pistettä)";
    }
}

File: IWD_RTYÖ/Pelaaja.js
class Pelaaja {
  constructor(nimi, pisteet) {
    this.pelaajaNimi = nimi;
    this.pelaajanPisteet = pisteet;
  }

  getPelaajaNimi() {
    return this.pelaajaNimi;
  }

  toString() {
    return `${this.pelaajaNimi} (${this.pelaajanPisteet} pistettä)`;
  }
}

module.exports = Pelaaja;
